src/cltkv1/utils/data_types.py

```diff
@@ -94,6 +103,25 @@ def sentences(self):
             for sentence in self.indices_tokens
         ]
 
+    @property
+    def tokens_list(self) -> List[str]:
```

Comment:
@free-variation when I make this `tokens`, it fails with some conflict in stanford.

Reply:
Odd.  I'll investigate.